icons/shapes.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15.6 10a.9.9 0 0 0 .9-.9c0-.2 0-.3-.1-.5l-3.6-6.2a.95.95 0 0 0-1.2-.3c-.1.1-.3.2-.3.3L7.6 8.7a.9.9 0 0 0 .8 1.3Z" />
+  <rect width="8" height="8" x="2" y="14" rx="2" />
+  <circle cx="18" cy="18" r="4" />
```

Comment:
I think maybe we should shrink this by 1 unit as such and better match similarly sized squares:

![image](https://github.com/lucide-icons/lucide/assets/17746067/0313897d-9cc1-4dbf-bf11-5d4c3d412010)


```suggestion
  <path d="M8.3 10a.7.7 0 0 1-.626-1.079L11.4 3a.7.7 0 0 1 1.198-.043L16.3 8.9a.7.7 0 0 1-.572 1.1Z" />
  <rect x="3" y="14" width="7" height="7" rx="1" />
  <circle cx="17.5" cy="17.5" r="3.5" />
```

Reply:
(And again, this version has actually been in my drafts for _ages_, do you work for Figma or something? 😆 )

Reply:
> do you work for Figma or something? 😆

@karsa-mistmere If I did, I don’t think they’d be too happy to discover I exclusively use Illustrator 😅